build/azure-pipelines/exploration-merge.yml

```diff
@@ -0,0 +1,26 @@
+trigger:
+  branches:
+    include: ['master']
+pr: none
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: "10.15.1"
+
+- script: |
+    set -e
+
+    git config user.email "andresse@microsoft.com"
```

Comment:
We should really make a generic alias for stuff like this...I'm still cleaning up stuff that Matt set up with his e-mail